must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -56,3 +56,15 @@ do
         { timeout 120 oc -n "${ns}" logs --all-containers "${pod}" &> "${LOG_DIR}"/"${pod}".log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
     done
 done
+
+# Add important notfications to a notifications.txt file at the root of the noobaa collection path
+UNMANAGED_NOOBAA_NOTIF="The noobaa deployed on this cluster is not managed by the storagecluster CR and will not react to configuration changes made to the CR"
```

Comment:
```suggestion
UNMANAGED_NOOBAA_NOTIF="The noobaa deployed on this cluster is not managed by the storagecluster CR and will not react to configuration changes"
```

Reply:
It will react to config changes on the Noobaa CR, I want this to be very specific.

Reply:
I changed the ending to: `... will not react to configuration changes made to storagecluster CR`